app/src/main/res/layouts/loginActivity/values/button_template_style.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <style name="button_template_style">
+        <item name="android:layout_width">336dp</item>
```

Comment:
Instead of fixed width, use fixed left/right margin to fit different size of screen better.